<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Perturbation</class>
 <widget class="QWidget" name="Perturbation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget_pert">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="tab_ligs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>Ligands</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QTableWidget" name="tableWidget_ligands">
                 <property name="gridStyle">
                  <enum>Qt::SolidLine</enum>
                 </property>
                 <property name="sortingEnabled">
                  <bool>true</bool>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="horizontalHeaderHighlightSections">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderCascadingSectionResizes">
                  <bool>true</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Exp. Affinity
 (kcal/mol)</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Exp. Error
(kcal/mol)</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Weight</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <italic>true</italic>
                  </font>
                 </property>
                 <property name="text">
                  <string>Build Perturbation Graph</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>Loading  Perburtaion Graph</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_2" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,2">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Graph File</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="button_graphLoad">
                    <property name="text">
                     <string>Browse</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>Generate  Perburtaion Graph</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="setWeightLabel">
                   <property name="text">
                    <string>Set Weights</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Please set on the above table</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="similarityMethodsLabel">
                   <property name="text">
                    <string>Similarity Methods</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QComboBox" name="similarity_method">
                   <item>
                    <property name="text">
                     <string>Tanimoto fingerprint</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>MACCS fingerprint</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Common Substructure</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="optimalCriteriaLabel">
                   <property name="text">
                    <string>Optimal criteria</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QComboBox" name="optimal_criteria">
                   <item>
                    <property name="text">
                     <string>A-optimal</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>G-optimal</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>D-optimal</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="numberOfCycleLabel">
                   <property name="text">
                    <string>Number of cycle</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QSpinBox" name="number_of_cycles"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="button_graphGen">
                 <property name="text">
                  <string>Build Perturbation Graph</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_pairs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>Pairs</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTableWidget" name="tableWidget_pairs">
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Ligand A</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Ligand B</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>sim. score</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>imgs/ArrowDouble.png</normaloff>imgs/ArrowDouble.png</iconset>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Ops</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="graph_frame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="graph_layout" stretch="0,0">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Perturbation Graph</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="button_row">
          <item>
           <widget class="QPushButton" name="button_graphRedo">
            <property name="text">
             <string>Redo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_graphUndo">
            <property name="text">
             <string>Undo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_graphAddEdge">
            <property name="text">
             <string>Add Edge</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_graphDelEdge">
            <property name="text">
             <string>Delete Edge</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_next_pert">
     <property name="text">
      <string>Confirm Pertubation Graph &amp; Go to Check Mapping </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
